pkg/simulator/terraform.go

```diff
@@ -47,8 +47,24 @@ func Terraform(wd, cmd, bucket string) (*string, error) {
 func InitIfNeeded(logger *zap.SugaredLogger, tfDir, bucket string) error {
 	logger.Debug("Terraform.InitIfNeeded() start")
 
+	logger.Info("Running terraform init")
+	_, err := Terraform(tfDir, "init", bucket)
+	if err != nil {
+		return errors.Wrap(err, "Error initialising terraform")
```

Comment:
If there's an error initialising Terraform, what actions does the user need to perform (if any at all; even if it is as simple as just re-running the command or troubleshooting)?

Reply:
This is a good point :) Currently it will output the terraform output (as well as a stack trace) and hope the user knows how to deal with things and this is not ideal - it's the source of some of the cryptic messages we've come across in other bugs raised.  I have a ticket to deal with the logging as part of the scenario discovery epic which I hope will improve things.  
1. There should be no stack trace (its not a critical failure of the simulator binary)
1. We should try to be smarter about detecting errors we can know about when running `terraform init`

The error behaviour does not change from master for this particular pull request but it's definitely something we need to tackle.